### PR TITLE
Update README for initial release

### PR DESCRIPTION
**Pull request by stonko1994**
## Description

This PR updates the README of this package in preparation for its initial public release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,83 @@
...
+source 'https://github.com/bitmovin/cocoapod-specs.git'
+
+pod 'BitmovinPlayerCore', '3.37.1'
+```
+
+### Using [Swift Package Manager](https://swift.org/package-manager/)
+[Swift Package Manager](https://swift.org/package-manager/) is a tool for managing the distribution of Swift frameworks. It integrates with the Swift build system to automate the process of downloading, compiling, and linking dependencies.
+
+#### Using Xcode
+To integrate using Xcode 13, open your Project file and specify it in `Project > Package Dependencies` using the following URL:
```
Comment (rolandkakonyi) [inline README.md]: We support Xcode 14.1 and above, so maybe worth mentioning 14 at least here.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,83 @@
...
+
+```
+https://github.com/bitmovin/player-ios-core.git
+```
+
+#### Using `Package.swift`
+To integrate using Apple's Swift Package Manager, add the following as a dependency to your `Package.swift` and replace `Version Number` with the desired version of the SDK.
+
+```swift
+.package(name: "BitmovinPlayerCore", url: "https://github.com/bitmovin/player-ios-core.git", .exact("Version Number"))
```
Comment (rolandkakonyi) [inline README.md]: The `.package(name:, url:, .exact())` API is deprecated, we should use the modern version.

See: https://developer.apple.com/documentation/packagedescription/package/dependency/package(name:url:_:)-6k3na

vs

https://developer.apple.com/documentation/packagedescription/package/dependency/package(url:exact:)

```suggestion
.package(url: "https://github.com/bitmovin/player-ios-core.git", exact: "Version Number")
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,83 @@
...
+```
+
+And then specify the `BitmovinPlayerCore` as a dependency of the desired target. Here is an example of a `Package.swift` file:
+
+```swift
+let package = Package(
+  ...
+  dependencies: [
+    ...
+    .package(name: "BitmovinPlayerCore", url: "https://github.com/bitmovin/player-ios.git", .exact("Version Number"))
```
Comment (rolandkakonyi) [inline README.md]: same here + please change the the URL to the core!

File: README.md [hunk 4]
```diff
@@ -1,8 +1,83 @@
...
+  ],
+  targets: [
+    .target(name: "<NAME_OF_YOUR_PACKAGE>", dependencies: ["BitmovinPlayerCore"])
```
Comment (rolandkakonyi) [inline README.md]: In this case we have to use this for proper access,
```suggestion
    .target(name: "<NAME_OF_YOUR_PACKAGE>", dependencies: [.product(name: "BitmovinPlayerCore", package: "player-ios-core")])
```

File: README.md (hunk 1 above)
Comment (stonko1994) [reply README.md]: [daf9195](https://github.com/bitmovin/player-ios-core/pull/1/commits/daf919524d4f38d6709fe42aaa8589d45dad90f2)

File: README.md (hunk 3 above)
Comment (stonko1994) [reply README.md]: [0ec8006](https://github.com/bitmovin/player-ios-core/pull/1/commits/0ec80069212fde0680eb6575d7ce83b0b81e7afc) + [19b83fd](https://github.com/bitmovin/player-ios-core/pull/1/commits/19b83fd7bd72bb31b0dc0146951cf4c9b07a7188)

File: README.md (hunk 4 above)
Comment (stonko1994) [reply README.md]: [19b83fd](https://github.com/bitmovin/player-ios-core/pull/1/commits/19b83fd7bd72bb31b0dc0146951cf4c9b07a7188)

File: README.md (hunk 2 above)
Comment (stonko1994) [reply README.md]: [19b83fd](https://github.com/bitmovin/player-ios-core/pull/1/commits/19b83fd7bd72bb31b0dc0146951cf4c9b07a7188)

**Review by rolandkakonyi — APPROVED** [review]

**Review by rolandkakonyi — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,8 +1,88 @@
...
+- [Samples](#samples)
+- [Documentation and Resources](#documentation-and-resources)
+
+The `BitmovinPlayerCore` package is the standalone `Player` without additional dependencies to other modules such as `BitmovinAnalytics`.
+
+## Installation
+To add the `BitmovinPlayerCore` SDK as a dependency to your project, you have three options: Using CocoaPods, Swift Package Manager or adding the SDK bundle directly.
+
+### Using [CocoaPods](https://cocoapods.org/)
+Add the following lines to the `Podfile` of your project and replace the `Version Number` with the desired version of the `BitmovinPlayerCore`. You can find all versions [here](https://github.com/bitmovin/cocoapod-specs/tree/master/Specs/BitmovinPlayerCore).
```
Comment (hawk23) [inline README.md:18]: Nit: 
```suggestion
Add the following lines to the `Podfile` of your project and replace the version number with the desired version of the `BitmovinPlayerCore`. You can find all versions [here](https://github.com/bitmovin/cocoapod-specs/tree/master/Specs/BitmovinPlayerCore).
```

File: README.md [hunk 6]
```diff
@@ -1,8 +1,88 @@
...
+
+Execute `pod repo update` first to add the new source and execute `pod install` after to install the new `BitmovinPlayerCore` dependency.
+
+```ruby
+source 'https://github.com/bitmovin/cocoapod-specs.git'
+
+pod 'BitmovinPlayerCore', '3.37.1'
```
Comment (hawk23) [inline README.md:25]: Nit: Will our CI update the version automatically on release?

File: README.md [hunk 7]
```diff
@@ -1,8 +1,88 @@
...
+  ]
+  ...
+)
+```
+
+##### Limitation
+Executing `swift build` from the command line is currently not supported. Open the Package in Xcode if you are developing another Package depending on `BitmovinPlayerCore`.
+
+### Add the Player SDK to the Project directly
+When using Xcode, go to the `General` page or your app target and add the SDK bundle (`BitmovinPlayerCore.xcframework`) under `Linked Frameworks and Libraries`. The latest SDK for iOS and tvOS can be downloaded [here](https://cdn.bitmovin.com/player/ios_tvos/3.37.1/BitmovinPlayerCore.zip).
```
Comment (hawk23) [inline README.md:70]: Nit: Will our CI update the link automatically on release?

File: README.md [hunk 8]
```diff
@@ -1,8 +1,88 @@
...
+
+## Getting Started
+### Configure the license key
+
+- Add your Bitmovin Player license key to the `Info.plist` file as `BitmovinPlayerLicenseKey`. Alternatively, you can also set the license key via the `PlayerConfig.key` property when creating a `Player` instance.
+
+    Your Bitmovin Player license key can be found under `Player -> Licenses` in the [Bitmovin Dashboard](https://bitmovin.com/dashboard).
```
Comment (hawk23) [inline README.md]:
```suggestion
    Your Bitmovin Player license key can be found under `Player -> Licenses` in the [Bitmovin Dashboard](https://dashboard.bitmovin.com).
```

File: README.md (hunk 5 above)
Comment (stonko1994) [reply README.md:18]: I think I would prefer the old version of it 🤔 (This is consistent with `player-ios` then)

File: README.md (hunk 8 above)
Comment (stonko1994) [reply README.md]: [ff98a3a](https://github.com/bitmovin/player-ios-core/pull/1/commits/ff98a3a6868443a0cadec937e882bf586c2fb956)

File: README.md (hunk 6 above)
Comment (stonko1994) [reply README.md:25]: Maybe not for the first release but we have that already in place for player-ios so we can easily adapt

**Review by hawk23 — APPROVED** [review]

**Review by melaniePlainer — APPROVED** [review]
Great work! 👏🏻

**Pull request merged**